docs/api/image-focal-point.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: ImageFocalPoint (Component)
+---
+
+The ImageFocalPoint component is the main file of the library. It is a wrapper of the HTML image element that allows you to drag and drop the focal point button on the image.
```

Comment:
The ImageFocalPoint component is the main file of the library. It is a wrapper of the HTML image element that allows you to drag and drop a focal point button on top of the image element.